src/EnergyPlus/OutputReportTabular.cc

```diff
@@ -12971,11 +12971,10 @@ void WriteThermalResilienceTablesRepPeriod(EnergyPlusData &state, int const peri
     }
 }
 
-template <int numOfColumn>
+template <int columnNum>
 void WriteResilienceBinsTable(EnergyPlusData &state,
-                              int const columnNum,
                               std::vector<int> const &columnHead,
-                              const std::array<Real64, numOfColumn> DataHeatBalance::ZoneResilience::*memberPtr,
+                              const std::array<Real64, columnNum> DataHeatBalance::ZoneResilience::*memberPtr,
                               Real64 const unitConvMultiplier)
 {
     std::vector<Real64> columnMax(columnNum, 0);
```

Comment:
This can be a `std::array<Real64, columnNum>` now, which is the real benefit of making this change.